XMonad/Layout/Magnifier.hs

```diff
@@ -85,7 +85,7 @@ import XMonad.StackSet
 -- <https://xmonad.org/TUTORIAL.html#customizing-xmonad the tutorial> and
 -- "XMonad.Doc.Extending#Editing_the_layout_hook".
 --
--- Magnifier supports some commands, see 'MagnifyMsg'.  To use them add
+-- Magnifier supports some commands, see 'MagnifyMsg'. To use them add
```

Comment:
This seems to be done consistently throughout the module, right? If so, it would strike me as more style than spelling mistake

Reply:
Fair, I'll revert that